web_disable_export_group/static/src/js/disable_export_group.js

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2016 Onestein
+   Copyright 2018 Tecnativa - David Vidal
+   License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
+odoo.define("web_disable_export_group", function(require) {
+"use strict";
+
+    var core = require("web.core");
+    var Widget = require("web.Widget");
+    var Sidebar = require("web.Sidebar");
+    var session = require("web.session");
+    var _t = core._t;
+
+    Widget.include({
+        init: function (parent) {
+            this._super(parent);
+            session.user_has_group('web_disable_export_group.group_export_data').then(function(can_export) {
```

Comment:
Can you move this to web.Session (session_init or session_bind), please? I think web.Session is more appropriate.